packages/rocketchat-slackbridge/tests/manual-tests.txt

```diff
@@ -0,0 +1,58 @@
+Config:
```

Comment:
I do not know how to implement unit tests inside Rock.Chat yet so I need to keep a script of manual tests I need to run each time